### PR TITLE
BL-1175 Normalize LCCN numbers

### PR DESCRIPTION
**Pull request by sensei100**
- Updates the normalize_lccn method to only return the normalized lccn numbers
- Adds tests 
- Substitutes an empty space for the # symbol because the normalizer doesn't handle that edge case

### CODE REVIEW

File: lib/cob_index/macros/custom.rb [hunk 1]
```diff
@@ -416,9 +416,9 @@ def normalize_issn
 
       def normalize_lccn
         Proc.new do |rec, acc|
-          orig = acc.dup
-          acc.map! { |x| StdNum::LCCN.normalize(x) }
-          acc << orig
+          acc.map! { |x|
+            formatted_x = x.gsub("#", " ")
+            StdNum::LCCN.normalize(formatted_x) }
           acc.flatten!
```
Comment (dkinzer) [inline lib/cob_index/macros/custom.rb:422]: You don't  need acc.flatten! anymore if you're removing `acc << orig` on line 421

**Comment by sensei100** [comment]
@dkinzer I removed it.
